### DISCUSSION TITLE
RichEditor curator picker

### DISCUSSION THREAD

**Discussion by njugunamwangi** [awcodes/filament-curator#446] [Q&A] [answered]
Is it possible to use the curator picker in place of image picker in the Filament Form RichEditor?

**Comment by awcodes** [comment, marked as answer]
Not possible without changes to the core editor. Honestly not even sure if the underlying Trix instance supports swapping it out though.

**Comment by njugunamwangi** [comment]
I ended up using Tiptap editor in place of Richeditor and configured curator in the media action. Thanks for the two

**Reply by Tsrgtm** [reply]
How did you do it?
please guide me

**Reply by njugunamwangi** [reply]
Do you have coth curator and tiptap editor installed?

**Reply by Tsrgtm** [reply]
Yes I have both already installed

**Reply by njugunamwangi** [reply]
publish the config for tiptal editor, in actions, comment the first media action then uncomment the second; to have something lile

    // 'media_action' => FilamentTiptapEditor\Actions\MediaAction::class,
    'media_action' => Awcodes\Curator\Actions\MediaAction::class,

**Reply by Tsrgtm** [reply]
Thanks brother it worked

**Reply by njugunamwangi** [reply]
Glad I could help, Happy coding!
